common/neon/block.py

```diff
@@ -65,8 +71,25 @@ def _from_sol_block(cls, raw: SolRpcBlockInfo) -> Self:
             block_time=raw.block_time,
             parent_slot=raw.parent_slot,
             parent_block_hash=EthBlockHash.from_raw(raw.parent_block_hash.to_bytes()),
+            priority_fee_percentiles=cls._calculate_priority_fee_stats(raw),
         )
 
+    @classmethod
+    def _calculate_priority_fee_stats(cls, sol_block: SolRpcBlockInfo) -> list[int]:
```

Comment:
Please rename to:
`_calc_cu_price_stat`

Reply:
done